library/std/src/io/util.rs

```diff
@@ -19,7 +19,7 @@ pub struct Empty;
 
 /// Constructs a new handle to an empty reader.
 ///
-/// All reads from the returned reader will return <code>[Ok]\(0)</code>.
+/// All reads from the returned reader will return `Ok(0)`.
```

Comment:
this removes the intra-doc link, is that intentional?

Reply:
Oops, I've re-added the the intra-doc links with newer syntax. 👍🏽